cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/join/JoinField.java

```diff
@@ -26,22 +25,34 @@
  * The name of a field and an optional alias to rename it to.
  */
 @Beta
-public class JoinField extends Field {
+public class JoinField {
```

Comment:
Please don't change existing API. It's already public, so it's better not to change it unless really needed.

Reply:
This is not in any CDAP version, this is just.a revert of [an earlier BQ Pushdown PR](https://github.com/cdapio/cdap/pull/13746) which added the Fields classes, which are no longer required.